security/acme-client/src/opnsense/mvc/app/models/OPNsense/AcmeClient/AcmeClient.xml

```diff
@@ -612,6 +613,16 @@
                 <dns_linode_key type="TextField">
                   <Required>N</Required>
                 </dns_linode_key>
+                <dns_loopia_api type="TextField">
+                  <Required>N</Required>
+                  <default>https://api.loopia.se/RPCSERV</default>
+                </dns_loopia_api>
+                <dns_loopia_user type="TextField">
+                  <Required>N</Required>
```

Comment:
In theory you could use my constraint to make it only mandatory when the dropdown is set to this service:

https://github.com/opnsense/plugins/blob/master/www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml#L474-L481

Reply:
Nice idea. However, we would need to implement this for **houndreds** of fields in the Acme plugin in order to be actually useful. I don't think this is absolutely necessary. I haven't seen a single support request where a user forgot to add the credentials for DNS-01.